fisco-bcos/benchmark/table_benchmark.cpp

```diff
@@ -0,0 +1,250 @@
+        for (int i = 0; i < count; ++i)
+        {
+            tableFactory->createTable(prefix + to_string(i), keyField, valueFields, true, origin);
+        }
+    };
+    auto insert = [&](const string& tableName, const string& value, int count) {
+        auto table = tableFactory->openTable(tableName);
+        for (int i = 0; i < count; ++i)
+        {
+            auto entry = table->newEntry();
+            entry->setField("value", value);
+            table->insert(to_string(i), entry, std::make_shared<AccessOptions>(origin, true));
+        }
+    };
+    auto select = [&](const string& tableName, int count) {
+        auto table = tableFactory->openTable(tableName);
+        for (int i = 0; i < count; ++i)
+        {
+            auto entries = table->select(to_string(i), table->newCondition());
+            if (entries->size() == 0)
+            {
+                cout << "empty key:" << i << endl;
+            }
+        }
+    };
+    auto update = [&](const string& tableName, const string& value, int count) {
+        auto table = tableFactory->openTable(tableName);
+        for (int i = 0; i < count; ++i)
+        {
+            auto entry = table->newEntry();
+            entry->setField("value", value);
+            table->update(to_string(i), entry, table->newCondition(),
+                std::make_shared<AccessOptions>(origin));
+        }
+    };
+    auto remove = [&](const string& tableName, int count) {
+        auto table = tableFactory->openTable(tableName);
+        for (int i = 0; i < count; ++i)
+        {
+            table->remove(
+                to_string(i), table->newCondition(), std::make_shared<AccessOptions>(origin));
+        }
+    };
+    cout << "<<<<<<<<<< no permission accounts " << endl;
+    performance("create Table", tables, [&]() { createTable("table", tables); });
+    string testTableName("table0");
+    performance("Table insert", keys, [&]() { insert(testTableName, value, keys); });
+    performance("Table select", keys, [&]() { select(testTableName, keys); });
+    performance("Table update", keys, [&]() { update(testTableName, value + "0", keys); });
+    performance("Table remove", keys, [&]() { remove(testTableName, keys); });
+
+
+    if (authorityAccounts != 0)
+    {
+        setPermission(SYS_TABLES, authorityAccounts);
+        cout << "<<<<<<<<<< add permission accounts " << authorityAccounts << endl;
+        performance("create Table", tables, [&]() { createTable("pTable", tables); });
+        testTableName = "pTable0";
+        setPermission(testTableName, authorityAccounts);
+        performance("Table insert", keys, [&]() { insert(testTableName, value, keys); });
+        performance("Table select", keys, [&]() { select(testTableName, keys); });
+        performance("Table update", keys, [&]() { update(testTableName, value + "0", keys); });
+        performance("Table remove", keys, [&]() { remove(testTableName, keys); });
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Return value of std::remove() ignored. Elements remain in container.](https://app.codacy.com/gh/FISCO-BCOS/FISCO-BCOS/pullRequest?prid=5208383)